addons/website_event_sms/data/sms_data.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<odoo><data noupdate="1">
+
+    <record id="event_sms.sms_template_data_event_reminder" model="sms.template">
```

Comment:
I don't think we are going to add a module just for a data ... and we can't do conditional display in event based on website_published existence because it is broken wiht the new qweb design in 15+ (no more "if-else" constructs). Told people we would like to use it.